lib/watir/elements/element.rb

```diff
@@ -146,9 +146,10 @@ def tag_name
     # @param [:shift, :alt, :control, :command, :meta] modifiers to press while clicking.
     #
 
-    def click(*modifiers)
+    def click(*modifiers, scroll_to: :center)
```

Comment:
Is it inconsistent that we only apply the `:scroll_to` when modifiers are applied? `#click` without modifiers seems to be the only mouse action that doesn't centre the element first. Applying the `:scroll_to` would make `#obscured?` simpler too - ie could be made similar to the other click methods.